test/server/utils_test.cc

```diff
@@ -0,0 +1,20 @@
+#include "source/server/utils.h"
+
+#include "test/test_common/utility.h"
+
+#include "gtest/gtest.h"
+
+namespace Envoy {
+namespace Server {
+namespace Utilities {
```

Comment:
this namespace is plural whereas elsewhere you named it just `Utility`... probably want to unify these

Reply:
Makes sense. Updated.